DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/register_map_and_streaming_interfaces/sample.json

```diff
@@ -0,0 +1,93 @@
+{
+  "guid": "69415BED-D452-449A-8F5A-DB01ACCE38DC",
+  "name": "register map and streaming interfaces",
+  "categories": ["Toolkit/oneAPI Direct Programming/DPC++ FPGA/Tutorials/Features/experimental"],
+  "description": "An Intel® FPGA tutorial demonstrating the usage of register_map and streaming interface controls",
+  "toolchain": ["icpx"],
+  "os": ["linux", "windows"],
+  "targetDevice": ["FPGA"],
+  "builder": ["ide", "cmake"],
+  "languages": [{"cpp":{}}],
+  "commonFolder": {
+    "base": "../../../..",
+    "include": [
+      "README.md",
+      "Tutorials/Features/experimental/register_map_and_streaming_interfaces",
+      "include"
+    ],
+    "exclude": []
+  },
+  "ciTests": {
+    "linux": [
+      {
+        "id": "fpga_emu",
+        "steps": [
+          "icpx --version",
+          "mkdir build",
+          "cd build",
+          "cmake ..",
+          "make fpga_emu",
+          "./register_map_and_streaming_interfaces.fpga_emu"
+        ]
+      },
+      {
+        "id": "fpga_sim",
+        "steps": [
+          "icpx --version",
+          "mkdir build",
+          "cd build",
+          "cmake ..",
+          "make fpga_sim",
+          "./register_map_and_streaming_interfaces.fpga_sim"
+        ]
+      },
+      {
+        "id": "report",
+        "steps": [
+          "icpx --version",
+          "mkdir build",
+          "cd build",
+          "cmake ..",
+          "make report"
+        ]
+      }
+    ],
+    "windows": [
+      {
+        "id": "fpga_emu",
+        "steps": [
+          "icpx --version",
+          "cd ../../..",
```

Comment:
For this to work I think that you would need ../../../..
You can try by hand these instructions to make sure they are correct